test/over_react/shared/map_proxy_tests.dart

```diff
@@ -83,7 +83,7 @@ void mapProxyTests(Map mapProxyFactory(Map proxiedMap)) {
     test('forEach', () {
       var callback = (key, value) {};
 
-      expect(proxy.forEach(callback), isNull);
+      expect(() => proxy.forEach(callback), isNull);
       verify(mock.forEach(callback));
     });
 
```

Comment:
We should add tests for the new Dart 2 Map members as well

Reply:
@greglittlefield-wf agreed, but currently, these tests are skipped in the DDC because they make use of mirrors.

Reply:
Yeah, at some point, we need to update these tests to use DDC-compatible mocks, or to not use mocks at all. 

We do need some kind of test coverage around these methods; we should at least stub out these tests with FIXMEs or something so that we add them by the time we release these changes as stable.